src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <div class="search-bar__container">
```

Comment:
Use semantic tags where possible. For example, if this `div` represents a main navigation or a distinct section of the page, consider using a `<nav>` or `<section>` element to improve your page SEO and help screen readers.